templates/haskell/package.yaml

```diff
@@ -0,0 +1,139 @@
+# Use this file as a reference when you create package.yaml
+# for your new package.
+# If you have multiple packages, you may wish to extract common
+# definitions into a separate file.
+# Why we need this template:
+# • harmless default extensions
+# • a good set of warnings
+# • metadata fields that are better be set
+# • some tricks (e. g. RTS options)
+name:                myproject
+# version is 0 by default to indicate that nothing is ready
+version:             0
```

Comment:
Why not 0.1.0.0 like `stack new` and `cabal init`?

Reply:
Hm, I think I never used `stack new` or `cabal init`. But I think that `0.1.0.0` should denote first released version, i. e. MVP or something like that. And `0` means "it's not ready and doesn't work yet". `0` is not supposed to be uploaded to Hackage. Usually a version is assigned when a release is made, so if you open a commit where version is `X`, it means that code there is exactly `X` or newer. So I prefer setting version 0.1.0 when first release is made.